app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+  def bootstrap_class_for_toast(type)
+    case type
+    when 'error'
```

Comment:
hibás jelszó esetén a devise `alert` típust használ. lehet ezt kellene nekünk is, hogy mindenhol jól nézen ki?